### PR TITLE
[fix][client] Fix client erroneous code comments

### PR DESCRIPTION
**Pull request by crossoverJie**
### Motivation

Fix incorrect comments.

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: pulsar-client-1x-base/pulsar-client-1x/src/main/java/org/apache/pulsar/client/api/ClientConfiguration.java [hunk 1]
```diff
@@ -158,7 +158,7 @@ public int getIoThreads() {
     }
 
     /**
-     * Set the number of threads to be used for handling connections to brokers <i>(default: 1 thread)</i>.
+     * Set the number of threads to be used for handling connections to brokers <i>(default is Runtime.getRuntime().availableProcessors())</i>.
```
Comment (RobertIndie) [inline pulsar-client-1x-base/pulsar-client-1x/src/main/java/org/apache/pulsar/client/api/ClientConfiguration.java:161]: The module `pulsar-client-1x` is deprecated. You can make your changes here: https://github.com/apache/pulsar/blob/c172a775fa0fc5762a9703669ed8ebcd2efbe042/pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java#L245-L263

Comment (crossoverJie) [reply pulsar-client-1x-base/pulsar-client-1x/src/main/java/org/apache/pulsar/client/api/ClientConfiguration.java]: I was careless, it has been corrected.

**Review by RobertIndie — APPROVED** [review]

**Review by AnonHxy — APPROVED** [review]

**Comment by AnonHxy** [comment]
LGTM

**Comment by tisonkun** [comment]
@crossoverJie CI failed. I've fixed for you this time. Please check the status report and fix by yourself the next time :)

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Merging...

Thanks for your contribution @crossoverJie!

**Pull request merged**